server/domain/item.go

```diff
@@ -0,0 +1,37 @@
+package domain
+
+import "github.com/google/uuid"
```

Comment:
should we be importing a package in an entity? I imagined that entities would be package free, instead we would translate to our own `UUID` type. This may be a nit pick

Reply:
The example I was following used this same pattern.  I don't think it's a huge deal (it's not logic that would change very frequently), so I think a dependency like this is fine.  We could just call it a `string` though and then have our translation layer just convert the uuid lib we're using to a string